ansible/roles/ansible_service_broker_setup/tasks/openshift.yml

```diff
@@ -83,10 +88,28 @@
         -p AUTO_ESCALATE={{ broker_auto_escalate|bool }}
         -p KEEP_NAMESPACE={{ broker_keep_namespace|bool }}
         -p KEEP_NAMESPACE_ON_ERROR={{ broker_keep_namespace_on_error|bool }}
-        -p DAO_TYPE={{ broker_dao_type }}
         {% if broker_kind is defined %}-p BROKER_KIND={{ broker_kind }}{% endif %}
         {% if broker_auth is defined %}-p BROKER_AUTH='{{ broker_auth }}'{% endif %}
 
+  - name: Include deployment steps for etcd when deploying older broker versions
+    include_tasks: broker_etcd_setup.yml
+    when:
+    - broker_tag == "v3.9" or "v3.9" or "v3.7" or "v3.6" in broker_tag
```

Comment:
I wanted to do something like:  
- "v3.9" or "v3.7" or "v3.6" in broker_tag

but had some syntax errors with that, seemed like I needed to start line with a variable.
Would like to come back to fix this.

Reply:
when you start with double quotes the yaml parser wants the line to end with double quotes too.

The way they suggest in ansible to do it is just wrap the whole thing in single quotes.

`'"v3.9" or "v3.7" or "v3.6" in broker_tag'`

Reply:
To add if it were on one line:

when: "v3.9"... it would give you the hint to put it in single quotes, but I've also noticed it's a little mote cryptic when you add it as you have.

Reply:
Thanks @jmontleon :) 
Will update to this in a bit.